packages/insomnia-cookies/src/cookies.ts

```diff
@@ -1,40 +1,38 @@
-const { CookieJar, Cookie } = require('tough-cookie');
+import { CookieJar, Cookie, CookieJSON } from 'tough-cookie';
 
 /**
  * Get a list of cookie objects from a request.jar()
- *
- * @param jar
  */
-module.exports.cookiesFromJar = function(jar) {
+export const cookiesFromJar = (cookieJar: CookieJar): Promise<CookieJSON[]> => {
+  console.log({ cookieJar });
```

Comment:
Should probably get rid of this, or have it be prefixed by `[cookies]` or similar

Reply:
this is in error from when I was debugging. good catch, actually. I'll remove soon.

Reply:
fixed in 8a9da26a0d5a67403282723bcc306ec9bd2ac0ab